app/javascript/dashboard/routes/dashboard/conversation/contact/ContactInfo.vue

```diff
@@ -92,17 +91,30 @@ export default {
       return ` ${this.contact.name}?`;
     },
   },
+  watch: {
+    'contact.id'(id) {
+      this.$store.dispatch('contacts/fetchContactableInbox', id);
+    },
+  },
+  mounted() {
+    const { id } = this.contact;
+    this.$store.dispatch('contacts/fetchContactableInbox', id);
+  },
```

Comment:
Instead of having this on both mounted and watch, we can make the watch `immediate`